src/app/pages/institutional-partnership/sections/promoting/promoting.tsx

```diff
@@ -17,17 +38,17 @@ export default function Promoting({
                 <div className="cards">
                     <div className="wide card">
                         {
-                            wideCards.map((item) =>
-                                <div className="wide-card-entry" key={item}>
+                            wideCards.map((item, index) =>
+                                <div className="wide-card-entry" key={index}>
```

Comment:
If these are always in the same order, maybe it's fine, but using the image might be a more stable key